Jenkinsfile

```diff
@@ -94,6 +95,8 @@ pipeline {
             steps {
                 sh 'docker login -u="${DOCKER_USR}" -p="${DOCKER_PSW}"'
                 sh 'build/run make -j\$(nproc) -C build/release build BRANCH_NAME=${BRANCH_NAME} GIT_API_TOKEN=${GIT_PSW}'
+                sh 'git status'
```

Comment:
do you want to do a `git diff` output too, so you know what the changes were, not just which files they were in?

Reply:
good idea thanks